tests/conftest.py

```diff
@@ -44,14 +44,6 @@ def web3() -> Web3:
     # web3 = Web3(HTTPProvider("http://localhost:8555"))
```

Comment:
Is this for ganache needed anymore? can we remove this commented block?

Reply:
Nope; removed.